### DISCUSSION TITLE
六圣召唤！

### DISCUSSION THREAD

**Discussion by OoTyleroO** [share121/inter-knot#1159] [其他]
![1723308186513](https://github.com/user-attachments/assets/e332957e-ce56-4bf9-a4b5-a83a6848939b)
入驻啦！

**Comment by 1451374815** [comment]
支持一下（王离月干年）喵

**Comment by wsffdg-123** [comment]
开门，三十人团！

**Comment by dogofvita** [comment]
关注七圣召唤枫达杯12强牢伯喵，谢谢喵，你知道的，我一直是i伯

**Comment by shenguojia123** [comment]
不认识，我打三国杀去了

**Comment by SSTV-10068** [comment]
您好，您的广告可以挂在SSTV上的广告位上喔~
SSTV广告位价格实惠，广告醒目吸人，[欢迎大家抢购！](https://github.com/share121/inter-knot/discussions/1065)

**Comment by lllliufeng** [comment]
碰到广告机器人了？(

**Comment by SSTV-10068** [comment]
您好，SSTV不是机器人，而是正规的新闻播报电视台喔~（只不过是没有营业执照罢了）（笑
